docs/conf.py

```diff
@@ -76,10 +76,10 @@
     except StopIteration:
         raise Exception(f"Could not find provider.yaml file for package: {PACKAGE_NAME}")
     PACKAGE_DIR = CURRENT_PROVIDER['package-dir']
-    PACKAGE_VERSION = 'master'
+    PACKAGE_VERSION = 'devel'
```

Comment:
Wait are we naming it `devel` or `main` ?

Reply:
It would good to have the same name in the docs as the branch name

Reply:
Branch was going to be main, I was thinking that "main" in the docs isn't clear enough though.

It appears we don't actually show this anywhere though, so it's probably moot?

Reply:
Can change it if you'd like

Reply:
Yeah since we don't use RTD, it does matter I guess